automation/test.sh

```diff
@@ -277,7 +277,9 @@ elif [[ $TARGET =~ multus.* ]]; then
 elif [[ $TARGET =~ genie.* ]]; then
   ginko_params="$ginko_params --ginkgo.focus=Genie|Networking|VMIlifecycle|Expose"
 elif [[ $TARGET =~ sriov.* ]]; then
-  ginko_params="$ginko_params --ginkgo.focus=SRIOV" 
+  ginko_params="$ginko_params --ginkgo.focus=SRIOV"
+elif [[ $TARGET =~ okd.* ]]; then
+  ginko_params="$ginko_params --ginkgo.skip=Multus|Genie|SRIOV|pinning"
```

Comment:
I don't think we need to skip multus on this lane it should work if not this is an issue

Reply:
I believe this one `should create a virtual machine with one interface with network definition from different namespace` still fails

Reply:
@SchSeba Can you create the PR to skip it on the OpenShift4

Reply:
Sure I can work on that

Reply:
@SchSeba Thanks!